README.md

```diff
@@ -99,29 +99,27 @@ The metrics generated by DVO can be scraped by anything that understands prometh
 oc process --local NAMESPACE='some-namespace' -f deploy/openshift/network-policies.yaml | oc create -f -
 ```
 
-## Enabling Checks
+## Configuring Checks
 
-In the validation engine file is a function called getDisabledChecks(). Comment out the check you would like to have available to the user base of DVO and push the change.
+DVO performs validation checks using kube-linter. The checks configuration is mirrored to the one for the kube-linter project. More information on configuration options can be found [here](https://github.com/stackrox/kube-linter/blob/main/docs/configuring-kubelinter.md), and a list of available checks  can be found [here](https://github.com/stackrox/kube-linter/blob/main/docs/generated/checks.md).
 
-```
-Location of file
-../pkg/validations/validation_engine.go 
-```
+To configure DVO with a different set of checks, create a ConfigMap in the cluster with the new checks configuration. An example of a configuration ConfigMap can be found [here](./deploy/openshift/configmap.yaml).
 
-Documentation of currently allowed checks can be found [here](./docs/checks.md)
+**constraint**: Currently, the configuration isn't continuously monitored and is only checked at startup. If a new set of checks is configured in a ConfigMap, the pod running DVO will need to be rebooted.
 
-## Disabling Checks
+### Enabling checks
 
-In the current state of DVO, all validation checks are performed with [kube-linter](https://github.com/stackrox/kube-linter).
+To enable all checks, set the `addAllBuiltIn` property to `true`. If you only want to enable individual checks, include them as a collection in the `include` property and leave `addAllBuiltIn` with a value of `false`.
 
-kube-linter supports functionality for [ignoring violations for a particular kubernetes object](https://github.com/stackrox/kube-linter/blob/main/docs/configuring-kubelinter.md#ignoring-violations-for-specific-cases). This functionality is also supported through DVO.
+The `include` property can work together with `doNotAutoAddDefaults` set to `true` in a whitelisting way. Only the checks collection passed in `include` will be executed.
 
-In the validation engine file is a function called getDisabledChecks(). Comment the check you would like to have disabled to the user base of DVO and push the change.
+### Disabling checks
 
-```
-Location of file 
-../pkg/validations/validation_engine.go 
-```
+To disable all checks, set the `doNotAutoAddDefaults` property to `true`. If you only want to disable individual checks, include them as a collection in the `exclude` property and leave `doNotAutoAddDefaults` with a value of `false`
+
+The `exclude` property takes precedence over the `include` property. If a particular check is in both collections, it will be excluded by default.
+
+The `exclude` property can work in conjunction with `addAllBuiltIn` set to `true` in a blacklisting fashion. All checks will be triggered and only the checks passed in `exclude` will be ignored.
```

Comment:
Maybe we could also say what the default configuration is (when the configmap doesn't exist).